addon/templates/components/ember-wormhole.hbs

```diff
@@ -0,0 +1,3 @@
+{{ember-wormhole-placeholder registry=registry as="head" dom=dom}}
```

Comment:
Great idea- I'll actually pass the `_firstNode` and `_lastNode`s directly. Seems to be working well.